dxr/plugins/python/__init__.py

```diff
@@ -1,4 +1,16 @@
 """Python plugin, using the ast and tokenize standard library modules"""
 
-from .indexers import FileToIndex, mappings
-from .filters import TypeFilter, FunctionFilter
+from dxr.plugins import Plugin
+
+from .indexers import mappings, TreeToIndex
+from .filters import BasesFilter, DerivedFilter, FunctionFilter, TypeFilter
+
+
+plugin = Plugin(
+    filters=[BasesFilter, DerivedFilter, FunctionFilter, TypeFilter],
+    tree_to_index=TreeToIndex,
+    mappings=mappings,
+    config_schema = {
+        'python_path': str
```

Comment:
You should probably use the AbsPath thing from config.py rather than str, so you'll be immune to working directory changes (and support unicode paths, and SPOT, and all that).